network/robus/src/robus.c

```diff
@@ -223,6 +223,13 @@ error_return_t Robus_SetTxTask(ll_service_t *ll_service, msg_t *msg)
         ack = ctx.rx.status.unmap;
         full_size++;
     }
+#ifndef VERBOSE_LOCALHOST
```

Comment:
I think this means that if we are in verbose localhost, timestamps will not work on localhost anymore.